lib/releaselib.sh

```diff
@@ -50,12 +51,10 @@ release::update_job_cache () {
     logecho -r " $job cache..."
   fi
 
-  run=$($GSUTIL cat $logroot/$job/latest-build.txt 2>/dev/null) || return
-
   for ((cnt=0;cnt<=$cache_limit;cnt++)); do
-    # Once we hit one that is set, break out
+    # Once we hit a JOB[$run] that is set (or run gets to 0), break out
     # the cache should be complete after that
-    [[ -n ${JOB[$run]} ]] && break
+    ((run==0)) || [[ -n ${JOB[$run]} ]] && break
```

Comment:
got it. I couldn't figure out where it JOB was getting prefilled; I missed the `source` line.